### PR TITLE
Fix for extreme D level term PID1 Horizon

### PR DESCRIPTION
**Pull request by borisbstyle**
Slightly modified code to change the behaviour of the highest and the lowest selectable value for Angle Level D term, which is used for Horizon Sensitivity

Flight tested through the entire Level D range

### CODE REVIEW

File: src/main/flight/pid.c [hunk 1]
```diff
@@ -680,12 +680,11 @@ static void pidRewrite(pidProfile_t *pidProfile, controlRateConfig_t *controlRat
 
         // PID D Level Term is used for Horizon Sensitivity. It is adjusted so the default value of 100 works pretty well.
         // Default Level D term of 100 equals to 80% sensitivity and 125 and above is 100% sensitivity. It is scaled to prevent too much truncating n integers
-        if(pidProfile->D8[PIDLEVEL] == 0){
-            horizonLevelStrength = 0;
-        } else if (pidProfile->D8[PIDLEVEL] >= 125){
-        	horizonLevelStrength = 100;
+
+        if (pidProfile->D8[PIDLEVEL] >= 125){
+            horizonLevelStrength = constrain((horizonLevelStrength - 100) + 100, 0, 100);
```
Comment (ledvinap) [inline src/main/flight/pid.c]: `(horizonLevelStrength - 100) + 100 == horizonLevelStrength`

**Comment by borisbstyle** [comment]
@ledvinap 
Range increased to be able to use 0-255 D. The more D you use more rate mode you get! With 255 I was able to get these very nice and slow flips/rolls, but still leveling in very narrow angle of the sticks.

**Comment by borisbstyle** [comment]
@hydra 
Why did this one fail checks?

**Comment by borisbstyle** [comment]
Why does my change affect CJMCU? Is that probably some other change what causes this?

**Comment by ledvinap** [comment]
it't just that CJMCU does not fit into FLASH ... not much you can do about it ..

**Pull request merged**